uat/custom-components/client-java-sdk/src/main/java/com/aws/greengrass/testing/mqtt5/client/grpc/GRPCLinkImpl.java

```diff
@@ -24,6 +25,12 @@ public class GRPCLinkImpl implements GRPCLink {
     private final GRPCDiscoveryClient client;
     private final GRPCControlServer server;
 
+    interface HalfsFactory {
```

Comment:
What is "Halfs" ?

Reply:
Should be halves

Reply:
Fixed in a21465f0f7894ca68f6a0efb31852366e0c430c3